py/picca/io.py

```diff
@@ -844,6 +627,8 @@ def read_from_pix(in_dir,
     Returns:
         List of read spectra for all the healpixs
     """
+    userprint('DeprecationWarning: this method will be deprecated.')
```

Comment:
wouldn't it be better to use `warnings.warn("this method will be deprecated.", DeprecationWarning)`. No show stopper, but might be good practice for the future